### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

**Description**

Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/google-android-fhir/actions/runs/6288135984/attempts/1#summary-17073322067

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/google-android-fhir/actions/runs/6288136028/attempts/1#summary-17073322270

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

Ignoring typos is always a possibility... 

**Type**
Choose one: Trivial Bug fix

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [ ] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [ ] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [ ] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ To contribute to the project, please see [Contributing](https://github.com/googl
 
 You can create a [GitHub issue](https://github.com/google/android-fhir/issues) for bugs and feature requests.
 
-In case you find any security bug, please do NOT create a Github issue. Instead, email us at <android-fhir-sdk-feedback@google.com>.
+In case you find any security bug, please do NOT create a GitHub issue. Instead, email us at <android-fhir-sdk-feedback@google.com>.
```
Comment (jsoref) [inline README.md:41]: brand

File: workflow-testing/src/main/java/com/google/android/fhir/workflow/testing/CqlBuilder.kt [hunk 2]
```diff
@@ -87,7 +87,7 @@ object CqlBuilder : Loadable() {
   }
 
   /**
-   * Assembles an ELM Library exported as a JSON in to a FHIRLibrary
+   * Assembles an ELM Library exported as a JSON into a FHIRLibrary
```
Comment (jsoref) [inline workflow-testing/src/main/java/com/google/android/fhir/workflow/testing/CqlBuilder.kt:90]: changes from `in to` to `into` tend to be fairly controversial

File: buildSrc/src/main/kotlin/FirebaseTestLabConfig.kt [hunk 3]
```diff
@@ -79,7 +79,7 @@ fun Project.configureFirebaseTestLabForMicroBenchmark() {
 }
 
 private fun FlankGradleExtension.commonConfigurationForFirebaseTestLab(project: Project) {
-  projectId.set("android-fhir-instrumeted-tests")
+  projectId.set("android-fhir-instrumented-tests")
```
Comment (vorburger) [inline buildSrc/src/main/kotlin/FirebaseTestLabConfig.kt:82]: Cool PR! This is the only change that might warrant a closer look from another pair of eyes... I'm not yet familiar enough with [Flank](https://flank.github.io/flank/) to be sure this (clearly misspelt, yes; but still) `projectId` is safe to change... it doesn't seem to be a [reference to anything else in this repo](https://github.com/search?q=repo%3Agoogle%2Fandroid-fhir%20android-fhir-instrumeted-tests&type=code) -  but in theory could be to something external, say if this was actually the name of a project configured in https://firebase.google.com/docs/test-lab/? The only way to know if this is safe, without spending too much time, is to have working CI which is either green or red. Let's wait for #2197 to be fixed, and then I'll approve running this; if it's green, I'll merge it.

Comment (omarismail94) [reply buildSrc/src/main/kotlin/FirebaseTestLabConfig.kt:82]: This is misspelt but the project name is actually named that

Comment (jsoref) [reply buildSrc/src/main/kotlin/FirebaseTestLabConfig.kt:82]: Sorry, to be clear, the project name itself includes the misspelling? If so, I'll obviously, drop this change.

Comment (vorburger) [reply buildSrc/src/main/kotlin/FirebaseTestLabConfig.kt:82]: @jsoref yeah, apparently. If you could revert this line, then I would be happy to merge this!

Comment (jingtang10) [reply buildSrc/src/main/kotlin/FirebaseTestLabConfig.kt]: This is unfortuante. @omarismail94 can this be fixed?

**Comment by jsoref** [comment]
@jingtang10: a commit close to https://github.com/jsoref/google-android-fhir/commit/0029254118a6a91032cdc3cb9f435c5258840277#diff-aa48ce5bca2bfbf528400261b9c5a206f60410e880f71a80499efbe87ea236b2 could be added to the repository, I'd be happy to make such a PR (I tend to make a couple of adjustments -- especially setting up a feature for use in forks to ease configuration burden for contributors, and I need to incorporate a way to improve the handling of dependabot triggered PRs). But I like to talk to project owners to give them a sense of how things will work -- there are a bunch of tunables and it helps for people to understand what will happen before it's deployed. I'm available on [Zulip](https://chat.fhir.org/#narrow/dm/656084-Josh-Soref) if that's helpful.

The main choices are:
* report a :x: status for the check (I generally do this, but one project opted out of this) -- this is configurable in the workflow file
* include a comment in the PR (I used to do this by default, but have mostly switched to relying on github status reports instead -- enabling the comment means giving the workflow `pull-requests: write` which people don't really like, and I agree) -- this is configurable in the workflow file
* the particulars of a footer to the report ([`advice.md`](https://github.com/jsoref/google-android-fhir/blob/0029254118a6a91032cdc3cb9f435c5258840277/.github/actions/spelling/advice.md) and [microsoft/terminal's implementation](https://github.com/microsoft/terminal/blob/main/.github/actions/spelling/advice.md) for some examples) that's appended at the end of reports when the tool finds problems -- this can be tweaked at any time in that file.
* any particular files to exclude ([`excludes.txt`](https://github.com/jsoref/google-android-fhir/blob/0029254118a6a91032cdc3cb9f435c5258840277/.github/actions/spelling/excludes.txt)) -- this file (or a split of it) can be tweaked at any time.

**Comment by jingtang10** [comment]
Thanks - please feel free to send a pr. We can talk about the specifics in the pr.

**Pull request merged**